packages/react/src/FilteredActionList/FilteredActionList.module.css

```diff
@@ -13,3 +13,21 @@
     }
   }
 }
+
+.SelectAllContainer {
+  display: flex;
+  align-items: center;
+  padding-block: var(--base-size-12);
```

Comment:
```suggestion
  padding-block: var(--base-size-4);
```

This is a bit nitpicky and I'm not sure if you're working with a design for this, but I think the padding is a little chonky. I wonder if we could swap to 4px. This could be a followup PR for sure!

<img width="362" height="465" alt="Padding" src="https://github.com/user-attachments/assets/b07e74a0-c581-4dc5-8eff-90b62c7c8895" />

Reply:
Oh yeah, I was going from the designs [here](https://github.com/github/primer/issues/5348) and [here](https://www.figma.com/design/lIPxmElkJmh4zKRnCMoxG2/Teams--Org---Enterprise-?node-id=4-36830&t=h2js9qvrVoxEbh1u-0)--they are both a little different. What do you think based on that? I could definitely make it a bit smaller!